### ISSUE TITLE
LaTeX -> Markdown Nested Enumerate Broken

### ISSUE THREAD

**Issue by rjturner** [jgm/pandoc#1774]
I am converting LaTeX to Markdown, and I found that nested enumerated lists do not correctly change the enumeration label from arabic to lowercase [as done by LaTeX](https://en.wikibooks.org/wiki/LaTeX/List_Structures#Nested_lists).

My input is `test.tex`:

```
\begin{enumerate}
    \item Item 1.
        \begin{enumerate}
            \item Item (a)
            \item Item (b)
        \end{enumerate}
    \item Item 2.
\end{enumerate}
```

Running `pandoc -f latex -t markdown test.tex` yields:

```
1.  Item 1.

    1.  Item (a)

    2.  Item (b)

2.  Item 2.
```

The sublist should be labeled with (a) and (b), not 1. and 2.

I hesitated to report this but then remembered that LaTeX to Markdown conversion is probably an uncommon use case (the analogous Markdown to LaTeX case works beautifully). Further, I could not find any reference to it in past issues.

**Comment by jgm** [comment]
I suppose this could be handled at the level of the LaTeX reader, which could try to assign list styles to sublists in a way that matches LaTeX's (NB. overridable) defaults.

**Comment by tyl012** [comment]
I also encountered the same problem, and it's been 10 years since the original post. It would be really helpful if anyone could solve this problem.

**Comment by jgm** [comment]
The default alternation in LaTeX is:
```
1.
  (a)
    i.
      A.
```

**Comment by tyl012** [comment]
Thank you for your reply, but I don't quite understand it (perhaps it's because I am not familiar with the way pandoc works). I will give an example of what I meant.

Here is a MWE in latex

> \documentclass{article}
> \begin{document}
> \begin{enumerate}
> \item item 1
> \item item 2
> \begin{enumerate}
> \item sub item 1
> \item sub item 2
> \begin{enumerate}
> \item ssub item 1
> \item ssub item 2
> \end{enumerate}
> \end{enumerate}
> \end{enumerate}
> \end{document}

using the command

> pandoc -s test.tex  -o test.md

the output in the markdown file is 


```
1.  item 1

2.  item 2

    1.  sub item 1

    2.  sub item 2

        1.  ssub item 1

        2.  ssub item 2
```

this will be render properly in github

> 1.  item 1
> 
> 2.  item 2
> 
>     1.  sub item 1
> 
>     2.  sub item 2
> 
>         1.  ssub item 1
> 
>         2.  ssub item 2

but when I use this md file in jupyterbook, I get

![Image](https://github.com/user-attachments/assets/2120ac4b-68e2-4095-89f7-5f49b1f048a0)

but I want to get something similar to the original pdf output by latex

![Image](https://github.com/user-attachments/assets/31d83a6b-14f5-403f-8ec8-9a8df1f64897)

I think this could be caused by the way how jupyterbook works, but I want to know is there a way to get the md output directly in the form 

```
1.  item 1

2.  item 2

    a.  sub item 1

    b.  sub item 2

        i.  ssub item 1

        ii.  ssub item 2
```

**Comment by jgm** [comment]
Yes, sorry, I did understand your request; I was just writing out the default markers used by LaTeX for my own record.

**Comment by tyl012** [comment]
Thank you for your reply! I have already put up a post asking about this there. I just wanted to know is it possible to do this in the pandoc output md file?

**Comment by jgm** [comment]
Well, we *could* change the LaTeX reader so that it uses LowerAlpha, etc. instead of DefaultStyle.
But I'm not sure that's a good idea.

You could also use a Lua filter to change the styles.

**Comment by tyl012** [comment]
Yeah, I agree with you that this seems to be a circuitous way to do it. Thanks again!
